.github/workflows/make-release.yml

```diff
@@ -10,7 +10,7 @@ name: "Make release"
 on:
   push:
     tags:
-      - "openssl-*"
+      - "openssl-**"
```

Comment:
Why is double asterisk needed?

Reply:
It's in the GitHub docs https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#running-your-workflow-only-when-a-push-of-specific-tags-occurs

Reply:
True! Then I close this PR. Sorry for the noise.